healthcare/api-client/v1/datasets/datasets_test.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2018 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+
+import datasets
+
+cloud_region = 'us-central1'
+project_id = os.environ['GOOGLE_CLOUD_PROJECT']
+
+dataset_id = 'test-dataset-{}'.format(int(time.time()))
+destination_dataset_id = 'test-destination-dataset-{}'.format(int(time.time()))
+keeplist_tags = 'PatientID'
+time_zone = 'UTC'
+
+
+def test_CRUD_dataset(capsys):
+    datasets.create_dataset(
+        project_id,
+        cloud_region,
+        dataset_id)
+
+    datasets.get_dataset(
+        project_id, cloud_region, dataset_id)
+
+    datasets.list_datasets(
+        project_id, cloud_region)
+
+    # Test and also clean up
+    datasets.delete_dataset(
+        project_id, cloud_region, dataset_id)
+
+    out, _ = capsys.readouterr()
+
+    # Check that create/get/list/delete worked
+    assert 'Created dataset' in out
+    assert 'Time zone' in out
+    assert 'Dataset' in out
+    assert 'Deleted dataset' in out
+
+
+def test_patch_dataset(capsys):
+    datasets.create_dataset(
+        project_id,
+        cloud_region,
+        dataset_id)
+
+    datasets.patch_dataset(
+        project_id,
+        cloud_region,
+        dataset_id,
+        time_zone)
+
+    # Clean up
+    datasets.delete_dataset(
+        project_id, cloud_region, dataset_id)
+
+    out, _ = capsys.readouterr()
+
+    # Check that the patch to the time zone worked
+    assert 'UTC' in out
+
+
+def test_deidentify_dataset(capsys):
```

Comment:
Please utilize a fixture.

Reply:
Used test_dataset fixture for the original dataset, but the new de-identified dataset that is created as part of the de-id operation must be deleted manually.